packages/mainsail/source/public-key.service.ts

```diff
@@ -5,14 +5,70 @@ import { abort_if, abort_unless } from "@ardenthq/sdk-helpers";
+import { ServiceProvider as CoreCryptoHashBcrypto } from "@mainsail/crypto-hash-bcrypto";
+import { ServiceProvider as CoreCryptoKeyPairEcdsa } from "@mainsail/crypto-key-pair-ecdsa";
+import { ServiceProvider as CoreCryptoSignatureSchnorr } from "@mainsail/crypto-signature-schnorr-secp256k1";
+import { Identifiers } from "@mainsail/contracts";
+import { milestones } from "./crypto/networks/devnet/milestones.js";
+import { network } from "./crypto/networks/devnet/network.js";
+
+import { ServiceProvider as CoreCryptoTransaction } from "@mainsail/crypto-transaction";
+import { ServiceProvider as CoreCryptoMultipaymentTransfer } from "@mainsail/crypto-transaction-multi-payment";
+import { ServiceProvider as CoreCryptoTransactionTransfer } from "@mainsail/crypto-transaction-transfer";
+import { ServiceProvider as CoreCryptoTransactionUsername } from "@mainsail/crypto-transaction-username-registration";
+import { ServiceProvider as CoreCryptoTransactionValidatorRegistration } from "@mainsail/crypto-transaction-validator-registration";
+import { ServiceProvider as CoreCryptoTransactionVote, VoteBuilder } from "@mainsail/crypto-transaction-vote";
+import { ServiceProvider as CoreCryptoTransactionValidatorResignation } from "@mainsail/crypto-transaction-validator-resignation";
+import { ServiceProvider as CoreFees } from "@mainsail/fees";
+import { ServiceProvider as CoreFeesStatic } from "@mainsail/fees-static";
 
 export class PublicKeyService extends Services.AbstractPublicKeyService {
 	readonly #config!: Interfaces.NetworkConfig;
+	readonly #app: Application;
+	#isBooted: boolean;
 
 	public constructor(container: IoC.IContainer) {
 		super(container);
 
 		this.#config = container.get(BindingType.Crypto);
+
+		this.#app = new Application(new Container());
+
+		this.#boot();
+	}
+
+	async #boot(): Promise<void> {
+		await Promise.all([
+			// this.#app.resolve(CoreValidation).register(),
+			// this.#app.resolve(CoreCryptoConfig).register(),
+			// this.#app.resolve(CoreCryptoValidation).register(),
+			// this.#app.resolve(CoreCryptoKeyPairEcdsa).register(),
+			// this.#app.resolve(CoreCryptoAddressBase58).register(),
+			// this.#app.resolve(CoreCryptoSignatureSchnorr).register(),
+			// this.#app.resolve(CoreCryptoHashBcrypto).register(),
+			// this.#app.resolve(CoreFees).register(),
+			// this.#app.resolve(CoreFeesStatic).register(),
+			// this.#app.resolve(CoreCryptoTransaction).register(),
+			// this.#app.resolve(CoreCryptoTransactionTransfer).register(),
+			// this.#app.resolve(CoreCryptoTransactionVote).register(),
+			// this.#app.resolve(CoreCryptoMultipaymentTransfer).register(),
+			// this.#app.resolve(CoreCryptoTransactionUsername).register(),
+			// this.#app.resolve(CoreCryptoTransactionValidatorRegistration).register(),
+			// this.#app.resolve(CoreCryptoTransactionValidatorResignation).register(),
+			// this.#app.resolve<Contracts.Crypto.PublicKeyFactory>(PublicKeyFactory),
```

Comment:
This line currently does nothing as it's missing a `register()` . I think you can just remove it because you already have ` this.#app.resolve(CoreCryptoKeyPairEcdsa).register()` above.